Tests/WorldCup/ScoreBoardTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Cheba\PhpUnit\Tests\WorldCup;
+
+use Cheba\PhpUnit\WorldCup\ScoreBoard;
+use PHPUnit\Framework\TestCase;
+
+class ScoreBoardTest extends TestCase
```

Comment:
No testing of negative path :
* passing negative score
* passing very long name

Reply:
Added testSetVeryLongTeamName, testSetNegativeScore and testSetEmptyTeamName.